### PR TITLE
oopsy: add more Abyssos savage

### PR DESCRIPTION
**Pull request by quisquous**

### CODE REVIEW

**Comment by quisquous** [comment]
I'm going to give this more testing before landing it.

File: ui/oopsyraidsy/damage_tracker.ts [hunk 1]
```diff
@@ -535,7 +536,7 @@ export class DamageTracker {
       const trigger: OopsyTrigger<OopsyData> = {
         id: key,
         type: 'GainsEffect',
-        netRegex: NetRegexes.gainsEffect({ effectId: id }),
+        netRegex: NetRegexes.gainsEffect({ effectId: id, ...playerTargetFields }),
```
Comment (quisquous) [inline ui/oopsyraidsy/damage_tracker.ts:539]: Gorgons get the petrification effect, so need to be excluded here.

**Comment by JLGarber** [comment]
> Being on the wrong side for Ptera Ixou

Is this an obfuscation-related thing? I know my log from last week has obfuscated debuffs on lines 26/30. Otherwise this *seems* like it should be straightforward?

**Comment by quisquous** [comment]
> > Being on the wrong side for Ptera Ixou
> 
> Is this an obfuscation-related thing? I know my log from last week has obfuscated debuffs on lines 26/30. Otherwise this _seems_ like it should be straightforward?

No, it's just a "I am tired and ran out of steam to finish" thing.

**Comment by JLGarber** [comment]
No worries. If you merge this now I'll look at that later tonight after raid maybe.

**Comment by JLGarber** [comment]
Commenting here for later, or if you want to add it:

```
EFFECTS:
CF7 -- Glossal Resistance Down (snake debuff, take on wing side)
CF8 -- Chelic Resistance Down (wing debuff, take on snake side)

ACTIONS:
787D -- Snake Ptera Ixou (name unknown, still obfuscated)
787E -- Wing Ptera Ixou (name unknown, still obfuscated)
```

**Comment by Akurosia** [comment]
787C/787D/787E are all just called `Ptera Ixou`

Full list of deobfuscated content of p6s can be found here: (there are also files with th eother translations for the rest of the raid containing all languages) 
https://discordapp.com/channels/551474815727304704/968569969103216651/1016082370971062273

**Pull request merged**